src/errors.ts

```diff
@@ -29,24 +29,24 @@ export function InvalidCryptoStoreError(reason) {
     Reflect.setPrototypeOf(instance, Reflect.getPrototypeOf(this));
     instance.reason = reason;
     instance.name = 'InvalidCryptoStoreError';
-    return instance;
 }
 
 InvalidCryptoStoreError.TOO_NEW = "TOO_NEW";
 
 InvalidCryptoStoreError.prototype = Object.create(Error.prototype, {
-  constructor: {
-    value: Error,
-    enumerable: false,
-    writable: true,
-    configurable: true,
-  },
+    constructor: {
+        value: Error,
+        enumerable: false,
+        writable: true,
+        configurable: true,
+    },
 });
 Reflect.setPrototypeOf(InvalidCryptoStoreError, Error);
```

Comment:
Same as above

Reply:
I wasn't entirely sure if that will work with just extending Error, as I knew there were some issues in the past with that that I have seen myself. Will do.